INSTALL.md

```diff
@@ -28,11 +28,13 @@ sudo apt-get install -y \
     pkg-config \
     runc \
     squashfs-tools \
+    squashfs-tools-ng \
```

Comment:
Just making sure we indeed want both squashfs-tools and squashfs-tools-ng here. (I _think_ we do, but just wanted to double-check.)

Reply:
Yep. We need `unsquashfs` / `mksquashfs` from `squashfs-tools`. Only `tar2sqfs` is used from `squashfs-tools-ng`